src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationQueryObjectType.cs

```diff
@@ -13,6 +18,31 @@ public LocalizationQueryObjectType(IStringLocalizer<LocalizationQueryObjectType>
 
             Field(x => x.Culture).Description(S["The culture for your content item."]);
             Field(x => x.LocalizationSet).Description(S["The localization set for your content item."]);
+
+            Field<ListGraphType<ContentItemInterface>, IEnumerable<ContentItem>>()
+                .Name("Localizations")
+                .Description(S["The localizations of the content item."])
+                .Argument<StringGraphType, string>("culture", "the culture of the content item")
+                .ResolveLockedAsync( async ctx =>
+                {
+                    var culture = ctx.GetArgument<string>("culture");
+                    var contentLocalizationManager = ctx.ResolveServiceProvider().GetService<IContentLocalizationManager>();
+
+                    if (culture != null)
+                    {
+                        var contentItems = new List<ContentItem>();
```

Comment:
you could save the list creation / adding here by using something like ...
```
var contentItem = await contentLocalizationManager.GetContentItemAsync(ctx.Source.LocalizationSet, culture);
return contentItem != null ? new[] { contentItem } : Enumerble.Empty<ContentItem>();
```

:)

Reply:
nice and clean. made the changes :)

Reply:
nice :) also can remove `var contentItems = new List<ContentItem>();` now :) think you missed it

Reply:
oops ;)